src/components/pagination/Pagination.vue

```diff
@@ -154,16 +162,29 @@ export default {
         ariaNextLabel: String,
         ariaPreviousLabel: String,
         ariaPageLabel: String,
-        ariaCurrentLabel: String
+        ariaCurrentLabel: String,
+        pageInput: {
+            type: Boolean,
+            default: false
+        },
+        pageInputPosition: String,
+        debouncePageInput: Number
```

Comment:
What's the need about debounce?

Reply:
This is to give the developer an option to put in their own denounce time depending on their needs that way a query, for example backend pagination, is not sent for every number input if the user is still inputting a large number.